packages/react/src/components/carbon-components-react/UIShell/HeaderMenu.js

```diff
@@ -125,6 +126,7 @@ class HeaderMenu extends React.Component {
       `${prefix}--masthead__megamenu__category-headline`,
       `${prefix}--masthead__megamenu__category-group`,
       `${prefix}--masthead__megamenu__view-all-cta`,
+      `${prefix}--masthead__megamenu__l0-nav`,
```

Comment:
adding this here so when user clicks on another L0 menu item, the overlay doesn't flash